app/assets/javascripts/tweets.js.coffee

```diff
@@ -0,0 +1,10 @@
+$ ->
+  $('.tweet-body').each ->
+    $(this).click ->
+      twttr.widgets.createTweet(
```

Comment:
Can we disable the `click` event before starting to load the widget? Sometimes it takes a good second to load and clicking on the tweet again will then load two widgets for the same tweet.

Reply:
Nice thanks